src/pudl/output/eia923.py

```diff
@@ -199,11 +232,43 @@ def fuel_receipts_costs_eia923(pudl_engine, freq=None,
 
     cols_to_drop = ['id', 'mine_id_pudl']
     frc_df = frc_df.drop(cols_to_drop, axis=1)
-
+    frc_df = pudl.helpers.convert_cols_dtypes(frc_df, data_source='eia')
+
+    if fill:
```

Comment:
I've been trying to only use the form `if thing is True` or `if thing is False` to avoid confusion with unexpected "True-ish" and "False-ish" values (like every non-empty string qualifies as `True`) since the `is` means it has to *actually* be identical to `True`. What do you think about these different kinds of tests?

Reply:
I've been following the pep-8 guidelines on this one:
["Don't compare boolean values to True or False using ==:"](https://www.python.org/dev/peps/pep-0008/)